application/controllers/admin/Authentication.php

```diff
@@ -258,7 +261,7 @@ public function newPassword()
             'errorExists' => $errorExists,
             'errorMsg' => $errorMsg,
             'randomPassword' => $randomPassword,
-            'validationKey' => $user->validation_key
+            'validationKey' => $validation_key
```

Comment:
Probably not part of this bug, but seems $validationKey is not sanitized.

https://github.com/LimeSurvey/LimeSurvey/pull/3895/files#diff-70d800faf45c8b44b4c4ae10435793c94be163ca4332e1701ab9c95812df4e7bR207

Or the standard anitization done by getParam('param') is enough?
Usually that's enough, but as we are then sending tha variable to the view, maybe we need something else as well?

Reply:
> No sanitization here, but you totally right : if it's invalid : it must be encoded in view. BUT : view didn't show it.

No real security issue : shown in input only of valid, and generated by LimeSurvey.